tests/integ/test_open_telemetry.py

```diff
@@ -54,6 +55,37 @@ def dict_exporter():
     dict_exporter.shutdown()
 
 
+def test_catch_error_during_action_function(session, dict_exporter):
+    df = session.sql("select 1/0")
+    with pytest.raises(SnowparkSQLException, match="Division by zero"):
+        df.collect()
+    spans = spans_to_dict(dict_exporter.get_finished_spans())
+    assert "collect" in spans
+    span = spans["collect"]
+    assert span.status.status_code == trace.status.StatusCode.ERROR
+    assert "Division by zero" in span.status.description
```

Comment:
just curious that does `span` or `span.status` contains other fields other than `description and `status_code`/do we care about other fields in the exception cases?

Reply:
It do contains other information like error code and detailded traceback, but for telemetry, we do not care about those, we actually only need the exception name, but since it is automatically collected by open telemetry, we don't need to remove or change them